src/dialog/createplaylistdialog.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <QDialog>
+#include <QLineEdit>
+#include <QCheckBox>
+
+class CreatePlaylistDialog : public QDialog
+{
+Q_OBJECT
+
+public:
+	CreatePlaylistDialog(QWidget *parent);
+
+	auto playlistName() -> std::string;
+	auto playlistDescription() -> std::string;
+	auto playlistPublic() -> bool;
+	auto playlistCollaborative() -> bool;
+
+private:
+	QLineEdit *name_editfield;
```

Comment:
Everything in `src/` more follows the style of Qt, camel case, so this should for example be `nameEditfield`.